clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -106,22 +107,29 @@ class ServerService private (
 
   private[service] def startup(): Future[Unit] = {
     import Elastic4sAutoDerivation._
+
+    val indexes = immutable.Seq(
+      ElasticsearchIndex.WgsUbam,
+      ElasticsearchIndex.Gvcf,
+      ElasticsearchIndex.WgsCram
+    )
+
     for {
       _ <- serverStatusDAO.setStatus(ServerStatusInfo.Starting)
-      _ <- persistenceDAO.initialize(
-        ElasticsearchIndex.WgsUbam,
-        ElasticsearchIndex.Gvcf
-      )
-      _ <- searchDAO.initialize()
+      _ <- persistenceDAO.initialize(indexes)
+      _ <- searchDAO.initialize(indexes)
       _ = logger.info("Recovering metadata from storage...")
-      Seq(recoveredUbamCount, recoveredGvcfCount) <- Future.sequence(
-        Seq(
-          recoverMetadata(ElasticsearchIndex.WgsUbam),
-          recoverMetadata(ElasticsearchIndex.Gvcf)
+      Seq(recoveredUbamCount, recoveredGvcfCount, recoveredCramCount) <- Future
+        .sequence(
+          Seq(
+            recoverMetadata(ElasticsearchIndex.WgsUbam),
+            recoverMetadata(ElasticsearchIndex.Gvcf),
+            recoverMetadata(ElasticsearchIndex.WgsCram)
```

Comment:
I can't just `map` over `indexes` here because `recoverMetadata` needs a bunch of implicits corresponding to its argument type, and making things generic breaks the implicit search. It'd be nice to move all the index-related implicits into the `ElasticsearchIndex` type as fields...